session.go

```diff
@@ -811,3 +784,45 @@ func finishBootstrap(store kv.Storage) {
 		log.Fatalf("finish bootstrap err %v", err)
 	}
 }
+
+const loadCommonGlobalVarsSQL = "select * from mysql.global_variables where variable_name in ('" +
```

Comment:
1. By executing update statement on this table.
2. Then there will be too many constant strings to concat, hard to read.